src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Shared/NavMenu.razor.css

```diff
@@ -7,6 +7,20 @@
     background-color: rgba(0,0,0,0.4);
 }
 
+.navbar {
+    width: 100%;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+a.navbar-brand {
+    padding-left: 3rem;
+    padding-right: 3rem;
+    white-space: normal;
+    text-align: center;
+    word-break: break-all;
+}
```

Comment:
I'm not sure these styles are needed if you leave the navbar-brand as it was previously.

Reply:
Reverted

Reply:
Oh, perhaps I should have waited to revert this until the navbar change is also reverted, leaving this unresolved for now so it still shows up